qemu/tests/qemu_guest_agent.py

```diff
@@ -885,6 +885,24 @@ def gagent_check_fsfreeze(self, test, params, env):
         # Finally, do something after thaw.
         self._action_after_fsthaw(test, params, env)
 
+    @error_context.context_aware
+    def gagent_check_thaw_unfrozen(self, test, params, env):
+        """
+        Thaw the unfrozen fs
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        try:
+            error_context.context("Verify if FS is thawed", logging.info)
+            expect_status = self.gagent.FSFREEZE_STATUS_THAWED
+            self.gagent.verify_fsfreeze_status(expect_status)
+        except guest_agent.VAgentFreezeStatusError:
+            self.gagent.fsthaw(check_status=False)
+        error_context.context("Thaw the unforzen FS", logging.info)
+        self.gagent.fsthaw(check_status=False)
```

Comment:
Should check the return value value.
such as {"return": 0} ,not {"return": 1} or others.

Reply:
Yes, self.gagent.fsthaw(check_status=False) will check it. If there is error, cmd() will raise error. And the output will be logged in file org.qemu.guest_agent.0-avocado-vt-vm1.log.
2018-01-08 01:01:44: {"execute": "guest-fsfreeze-thaw"}                          
2018-01-08 01:01:44: {"return": 0}

Reply:
self.gagent.fsthaw(check_status=False) will not check if the value is 0 or others.
{ "execute": "guest-fsfreeze-freeze"}
{"return": 4}
{"execute":"guest-fsfreeze-thaw"}
{"return": 4}
{"execute":"guest-fsfreeze-thaw"} ---thaw the unfreezon fs.
{"return": 0}

Reply:
Yes, you are right. Updated, pls review again.